### ISSUE TITLE
@nrwl/remix generators always fail

### ISSUE THREAD

**Issue by donocode** [nrwl/nx-labs#145]
### Description

I was trying to add a new remix app to an nx workspace as a test, but it would fail every time with the error:

```
TypeError: (0 , project_configuration_1.shouldDefaultToUsingStandaloneConfigs) is not a function
```

I tried every available remix generator using the nx extension in vscode and the dry runs would all trigger the same error.

To be sure I tried generating a new workspace with the remix preset, it also hit the same error. Here's the full output with `--verbose`:

```
npx create-nx-workspace@latest remix-2 --preset=@nrwl/remix --project=demo --verbose

 >  NX   Let's create a new workspace [https://nx.dev/getting-started/intro]

✔ Enable distributed caching to make your CI faster · No

 >  NX   Nx is creating your v15.7.0 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "npm install" several times. Please wait.

✔ Installing dependencies with npm
✔ Nx has successfully created the workspace: remix-2.

>  NX  Generating @nrwl/remix:preset


 >  NX   (0 , project_configuration_1.shouldDefaultToUsingStandaloneConfigs) is not a function


TypeError: (0 , project_configuration_1.shouldDefaultToUsingStandaloneConfigs) is not a function
    at getWorkspaceLayout (/home/nd/work/nx-test/remix-2/node_modules/@nrwl/devkit/src/utils/get-workspace-layout.js:25:96)
    at normalizeOptions (/home/nd/work/nx-test/remix-2/node_modules/@nrwl/remix/src/generators/application/lib/normalize-options.js:6:57)
    at /home/nd/work/nx-test/remix-2/node_modules/@nrwl/remix/src/generators/application/application.impl.js:10:66
    at Generator.next (<anonymous>)
    at /home/nd/work/nx-test/remix-2/node_modules/tslib/tslib.js:167:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/home/nd/work/nx-test/remix-2/node_modules/tslib/tslib.js:163:16)
    at default_1 (/home/nd/work/nx-test/remix-2/node_modules/@nrwl/remix/src/generators/application/application.impl.js:9:20)
    at /home/nd/work/nx-test/remix-2/node_modules/@nrwl/remix/src/generators/preset/preset.impl.js:14:65
    at Generator.next (<anonymous>)

 >  NX   Failed to apply preset: @nrwl/remix

   See above
```

### Replication
1. Create a new workspace with `npx create-nx-workspace@latest remix --preset=@nrwl/remix --project=demo`
2. Observe error

**Comment by donocode** [comment]
I found [the code that is breaking in @nrwl/devkit](https://github.com/nrwl/nx/blob/54c2169d77ca47a0723521b873e0c0ba37904a23/packages/devkit/src/utils/get-workspace-layout.ts#L35), however it was removed on January 23rd 2023. This change was published in version 15.7.0 of @nrwl/devkit so it isn't used by @nrwl/remix.

I guess some other internal changes that removed the export of this function were published in a minor version update somewhere and this mismatch of versions causes the problem.

**Comment by philipjfulcher** [comment]
You should see a new version of the `@nrwl/remix` plugin soon which resolves this.
